feat(ui): Implement Forgot password page

thanghq-zinza opened the pull request:
Add Forgot password page
Implement validation and fake API call

thanghq-zinza committed:
feat(ui): Implement Forgot password page

Add Forgot password page
Implement validation and fake API call

hungnp194 commented on src/app/(authPage)/forgot-password/page.tsx:
```diff
@@ -1,7 +1,117 @@
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { useRouter } from 'next/navigation';
+interface FormData {
+  email: string;
+}
+const schema = yup
```
```suggestion
import { useRouter } from 'next/navigation';

interface FormData {
  email: string;
}

const schema = yup
```

Các block code không liên quan đến nhau thì cách 1 dòng

hungnp194 commented on src/app/(authPage)/forgot-password/page.tsx:
```diff
@@ -1,7 +1,117 @@
+  const [loading, setLoading] = useState(false);
+  const [success, setSuccess] = useState(true);
+
+  const obSubmit = (data: FormData) => {
```
typo

thanghq-zinza committed:
fix: Add spacing between code blocks

Add 1 line between code blocks

SMALL CHANGE: typo oBSumbit to onSubmit in Forgot page file
